build.sbt

```diff
@@ -586,6 +594,63 @@ lazy val testJunitRunnerTests = project.module
         .value
   )
 
+lazy val testJunitEngine = project.module
+  .in(file("test-junit-engine"))
+  .settings(stdSettings("zio-test-junit-engine"))
```

Comment:
What's the rationale behind creating a new module instead of updating the existing one?

Reply:
This is different. Test-junit implements a JUnit 4 Runner that allows "junit-vintage" engine (compatibility layer to run JUnit 3 and 4 tests with the JUnit 5 platform) to discover the tests that extend this class. In this case, it's a new JUnit test engine able to discover by itself the ZIO tests